packages/mermaid/src/docs/intro/index.md

```diff
@@ -1,34 +1,37 @@
 
 > If you are familiar with Markdown you should have no problem learning [Mermaid's Syntax](n00b-syntaxReference.md).
 
 <img src="/header.png" alt="" />
 
-[![Build CI Status](https://github.com/mermaid-js/mermaid/actions/workflows/build.yml/badge.svg)](https://github.com/mermaid-js/mermaid/actions/workflows/build.yml) [![NPM](https://img.shields.io/npm/v/mermaid)](https://www.npmjs.com/package/mermaid) [![npm minified gzipped bundle size](https://img.shields.io/bundlephobia/minzip/mermaid)](https://bundlephobia.com/package/mermaid) [![Coverage Status](https://coveralls.io/repos/github/mermaid-js/mermaid/badge.svg?branch=master)](https://coveralls.io/github/mermaid-js/mermaid?branch=master) [![CDN Status](https://img.shields.io/jsdelivr/npm/hm/mermaid)](https://www.jsdelivr.com/package/npm/mermaid) [![NPM](https://img.shields.io/npm/dm/mermaid)](https://www.npmjs.com/package/mermaid) [![Join our Slack!](https://img.shields.io/static/v1?message=join%20chat&color=9cf&logo=slack&label=slack)](https://join.slack.com/t/mermaid-talk/shared_invite/enQtNzc4NDIyNzk4OTAyLWVhYjQxOTI2OTg4YmE1ZmJkY2Y4MTU3ODliYmIwOTY3NDJlYjA0YjIyZTdkMDMyZTUwOGI0NjEzYmEwODcwOTE) [![Twitter Follow](https://img.shields.io/twitter/follow/mermaidjs_?style=social)](https://twitter.com/mermaidjs_)
+<p class="space-x-2 flex items-center"><a href="https://github.com/mermaid-js/mermaid/actions/workflows/build.yml" target="_blank" rel="noreferrer"><img src="https://github.com/mermaid-js/mermaid/actions/workflows/build.yml/badge.svg" alt="Build CI Status"></a><a href="https://www.npmjs.com/package/mermaid" target="_blank" rel="noreferrer"><img src="https://img.shields.io/npm/v/mermaid" alt="NPM"></a><a href="https://bundlephobia.com/package/mermaid" target="_blank" rel="noreferrer"><img src="https://img.shields.io/bundlephobia/minzip/mermaid" alt="npm minified gzipped bundle size"></a><a href="https://coveralls.io/github/mermaid-js/mermaid?branch=master" target="_blank" rel="noreferrer"><img src="https://coveralls.io/repos/github/mermaid-js/mermaid/badge.svg?branch=master" alt="Coverage Status"></a><a href="https://www.jsdelivr.com/package/npm/mermaid" target="_blank" rel="noreferrer"><img src="https://img.shields.io/jsdelivr/npm/hm/mermaid" alt="CDN Status"></a><a href="https://www.npmjs.com/package/mermaid" target="_blank" rel="noreferrer"><img src="https://img.shields.io/npm/dm/mermaid" alt="NPM"></a><a href="https://join.slack.com/t/mermaid-talk/shared_invite/enQtNzc4NDIyNzk4OTAyLWVhYjQxOTI2OTg4YmE1ZmJkY2Y4MTU3ODliYmIwOTY3NDJlYjA0YjIyZTdkMDMyZTUwOGI0NjEzYmEwODcwOTE" target="_blank" rel="noreferrer"><img src="https://img.shields.io/static/v1?message=join%20chat&amp;color=9cf&amp;logo=slack&amp;label=slack" alt="Join our Slack!"></a> <a href="https://twitter.com/mermaidjs_" target="_blank" rel="noreferrer"><img src="https://img.shields.io/twitter/follow/mermaidjs_?style=social" alt="Twitter Follow"></a></p>
```

Comment:
Let's stick with markdown?
I remember breaking up this into different lines, that should be done.

Reply:
This was the reason why I changed it to HTML. I edited another line at the page (did not touch this) and badges started to occupy multiple lines instead of one. That's why I converted it to HTML and added `space-x-2 flex items-center` classes to it. The result is consistent and the badges will occupy only one line. I think this is desired behavior.

Reply:
Markdown adds `p` tag and we dont have control over one `p` tag. That is why I added HTML

Reply:
Current Markdown turns into this
![image](https://github.com/mermaid-js/mermaid/assets/2842432/523f6991-ae35-4852-b568-a92bb589127f)
That is a lucky coincidence that current page works. If you have working solution with Markdown, let me know

Reply:
Ah, the reason why this doesn't work is because the custom CSS in https://github.com/mermaid-js/mermaid/blob/0b8bb71d4355820e548de8ec8ae3c3b3a712a3ef/packages/mermaid/src/docs/intro/index.md?plain=1#L233 no longer works.

You can modify it to be `#about-mermaid + p + blockquote + img + p` and it should work (use inspect element to figure out that you're finding the `<h1 id="about-mermaid">` tag, then skipping `<p>`, `<blockquote>`, `<img>`, and then you're only modifying the next `<p>`

![image](https://github.com/mermaid-js/mermaid/assets/19716675/269e2ea1-b0ca-4f25-a46b-8092b186ce34)

It's probably worth putting a comment here too, since this behavior is pretty unintuitive.

```suggestion
<!--
  If the CSS for these shields breaks, you need to modify the Vitepress CSS at the end of this page,
  see discussion in https://github.com/mermaid-js/mermaid/pull/4596#discussion_r1291868884
-->

[![NPM](https://img.shields.io/npm/v/mermaid)](https://www.npmjs.com/package/mermaid)
[![Build CI Status](https://github.com/mermaid-js/mermaid/actions/workflows/build.yml/badge.svg)](https://github.com/mermaid-js/mermaid/actions/workflows/build.yml)
[![npm minified gzipped bundle size](https://img.shields.io/bundlephobia/minzip/mermaid)](https://bundlephobia.com/package/mermaid)
[![Coverage Status](https://codecov.io/github/mermaid-js/mermaid/branch/develop/graph/badge.svg)](https://app.codecov.io/github/mermaid-js/mermaid/tree/develop)
[![CDN Status](https://img.shields.io/jsdelivr/npm/hm/mermaid)](https://www.jsdelivr.com/package/npm/mermaid)
[![NPM Downloads](https://img.shields.io/npm/dm/mermaid)](https://www.npmjs.com/package/mermaid)
[![Join our Slack!](https://img.shields.io/static/v1?message=join%20chat&color=9cf&logo=slack&label=slack)](https://join.slack.com/t/mermaid-talk/shared_invite/enQtNzc4NDIyNzk4OTAyLWVhYjQxOTI2OTg4YmE1ZmJkY2Y4MTU3ODliYmIwOTY3NDJlYjA0YjIyZTdkMDMyZTUwOGI0NjEzYmEwODcwOTE)
[![Twitter Follow](https://img.shields.io/badge/Social-mermaidjs__-blue?style=social&logo=twitter)](https://twitter.com/mermaidjs_)
```

Reply:
Yes, I tried this, but I dont like extra marging around it and I dont like relying on something fragile as a custom css line that fixes behavior by addressing specific tag compositions instead of css classes.

Reply:
Put div with css class around and the rest is markdown. Counting on the document's structure is not reliable enough.

Reply:
> Put div with css class around and the rest is markdown

Awesome, I'm going to mark this as resolved, since I think your fix works pretty well! Sid, I know this is your conversation, so feel free to un-resolve this if you disagree.